swervelib/ctre/KrakenX60DriveControllerFactoryBuilder.java

```diff
@@ -15,12 +15,15 @@ public final class KrakenX60DriveControllerFactoryBuilder {
 
     private static final int CAN_TIMEOUT_MS = 250; 
     private static final int STATUS_FRAME_GENERAL_PERIOD_MS = 250;
+    private static final double ENCODER_CPR = 1;
+    private double encoderDistancePerPulse;
 
     private double nominalVoltage = Double.NaN; 
     private double currentLimit = Double.NaN;
 
-    public KrakenX60DriveControllerFactoryBuilder withVoltageCompensation(double nominalVoltage) {
+    public KrakenX60DriveControllerFactoryBuilder withVoltageCompensation(double nominalVoltage, int wheelDiameter) {
```

Comment:
why is this here? wheel diameter has nothing to do with voltage compensation. where is that we set the encoder distance per pulse (or equivalent) for the neo equivalent to this class?

Reply:
For the NEO equivalent, it happens in the constructor:

            double positionConversionFactor = Math.PI * moduleConfiguration.getWheelDiameter()
                    * moduleConfiguration.getDriveReduction();

so it gets the wheel diameter from the moduleConfiguration in the constructor, which makes a lot more sense than doing it here